upstream/CVE-2019-11250.json

```diff
@@ -0,0 +1,41 @@
+{
+  "id": "CVE-2019-11250",
+  "modified": "2019-08-08T02:03:04Z",
+  "published": "2019-08-08T02:03:04Z",
+  "summary": "Bearer tokens are revealed in logs",
+  "details": "The Kubernetes client-go library logs request headers at verbosity levels of 7 or higher. This can disclose credentials to unauthorized users via logs or command output. Kubernetes components (such as kube-apiserver) prior to v1.16.0, which make use of basic or bearer token authentication, and run at high verbosity levels, are affected.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.0/AV:L/AC:H/PR:L/UI:N/S:U/C:H/I:N/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "0"
```

Comment:
It is not mentioned when this issue was introduced. We assume it's from the beginning at the moment, and will update if it's wrong.